packages/components/src/date-time/README.md

```diff
@@ -60,3 +60,10 @@ Whether we use a 12-hour clock. With a 12-hour clock, an AM/PM widget is display
 
 - Type: `bool`
 - Required: No
+
+### isInvalidDay
+
+A function with a Date object representing a day as an argument returning a Boolean to signify if the day is valid or not. This function is passed to react-dates' `isOutsideRange` prop.
```

Comment:
>This function is passed to react-dates' `isOutsideRange` prop.

But:

1. It's not the function we pass. We pass a wrapped function which handles the conversion from Moment to Date in the callback.
2. This component is an abstraction. The consumer needn't and shouldn't know that `react-dates` exists, largely because we don't want to make the commitment that it will always be the underlying implementation. Otherwise we put ourselves in a bind for future maintenance if/when we decide to go a different direction.

```suggestion
A callback function which receives a Date object representing a day as an argument, and should return a Boolean to signify if the day is valid or not.
```